test/functional/test_framework/test_framework.py

```diff
@@ -14,6 +14,7 @@
 import sys
 import tempfile
 import time
+import traceback
```

Comment:
Here: https://github.com/bitcoin/bitcoin/blob/9c248e39f2807a7f89e555e99cc55cb2de1ad335/test/functional/test_framework/test_framework.py#L194

I must have missed out this import in a previous commit. Makes sense to fix it in this cleanup commit.